### ISSUE TITLE
Can we add the possibility for circle annotation?

### ISSUE THREAD

**Issue by ron-weiner** [wkentaro/labelme#129]
Hey!
Thanks for the great tool again!

I sought for the possibility to make circle annotations, i find it very useful because in some cases there is a perfect circle we wish to annotate and its less productive and less accurate to mark it all with lots of dots..

Is there some existing solution with labelme? is it part of the pipeline?

thanks

**Comment by wkentaro** [comment]
This is not possible with current labelme, and I think adding this feature is very helpful.
There is an issue of suggestion about **rectangle** annotation and that would be great if both annotations are supported in addition to the contour annotation.

**Comment by jbvo** [comment]
Question: would these kinds of shapes be converted to polygons under the hood?

**Comment by wkentaro** [comment]
@jbvo Yeah, I think so.

**Comment by velaia** [comment]
I just found out: The circle does not get translated into a shape when converting to COCO format. I ran the labelme2coco.py script. The script ended up converting only the center of the circle (x, y) and the chosen dot on the circle itself (x, y coordinates).
I've tested the VGG Image Annotator (VIA) from http://www.robots.ox.ac.uk/~vgg/software/via/ and that one seems to convert the circles properly into a shape when exporting to COCO format.
